internal/cuckoo/filter_test.go

```diff
@@ -0,0 +1,170 @@
+package cuckoo
```

Comment:
we use build tags to distinguish unit and integration tests

Reply:
Yep, will fix.